pkg/model/model.go

```diff
@@ -21,8 +22,8 @@ type Component struct {
 	DeletedAt soft_delete.DeletedAt `json:"-"`
 	Name      string                `json:"name"`
 	Type      string                `json:"type"`
-	Status    bool                  `json:"status"`
-	PublicURL string                `json:"public_url"`
+	Status    bool                  `json:"status" gorm:"-"`
```

Comment:
Why you check this field to `gorm:"-"`? Can you explain about it more for me? 🙇🏼‍♂️